plugin/src/main/java/com/denizenscript/denizen/objects/PolygonTag.java

```diff
@@ -696,6 +714,9 @@ public static void register() {
         // @mechanism PolygonTag.add_corner
         // @description
         // Returns a copy of the polygon, with the specified corner added to the end of the corner list.
+        // @example
+        // # Notes a new polygon with a new corner added. If the new corner has a location of "10,66,-2", and "my_polygon" has corners "0.0,0.0", "7.0,7.0", and "-5.0,6.0", then "my_new_polygon" will have corners  "0.0,0.0", "7.0,7.0", "-5.0,6.0", and "10.0,-2.0".

```

Comment:
okay now this is too far in the opposite direction.
It needs newlines, just not in such awkward places as you had before.
Right after a `.` or `,` is often a good spot, as a loose heuristic.
Lines of length ~120 characters should be uncommon, and should never go past around ~180 characters.

Reply:
Whoops, my bad for misunderstanding 😆 
Fixed :)